vulture_os/toolkit/api_parser/sentinel_one_singularity_mobile/sentinel_one_singularity_mobile.py

```diff
@@ -0,0 +1,160 @@
+        self.session = None
+
+        self.host = data["sentinel_one_singularity_mobile_host"]
+        self.client_id = data["sentinel_one_singularity_mobile_client_id"]
+        self.__secret = data["sentinel_one_singularity_mobile_client_secret"]
+
+    def get_access_token(self):
+        if not self.__access_token:
+            try:
+                url = f"https://{self.host}/api/auth/v1/api_keys/login"
+                data = {
+                    "clientId": self.client_id,
+                    "secret": self.__secret,
+                }
+                response = requests.post(url, json=data, headers=self.HEADERS)
+                response.raise_for_status()
+            except requests.exceptions.HTTPError as err:
+                logger.debug(err, extra={'frontend': str(self.frontend)})
+                exit(0)
+            else:
+                response_json = response.json()
+                self.__access_token = response_json["accessToken"]
+        return self.__access_token
+
+    def _connect(self):
+        if self.session is None:
+            self.session = requests.Session()
+            self.session.headers.update(self.HEADERS)
+            self.session.headers.update({"Authorization": f"Bearer {self.get_access_token()}"})
+        return True
+
+    def get_logs(self, since, to):
+        if self.session is None:
+            self._connect()
+
+        def convert(x): return x.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + 'Z'
+        url = f"https://{self.host}/api/threats/public/v1/threats"
+
+        params = {
+            "page": 0,
+            "size": self.SIZE,
+            "after": convert(since),  # >=
+            "before": convert(to),    # <=
+            "sort": "timestamp,asc",
+            "module": "ZIPS"
+        }
+
+        response = self.session.get(url, params=params)
+
+        resp = response.json()
+        logger.debug(f"url: {url}\nparams: {params}", extra={'frontend': str(self.frontend)})
+
+        if "content" not in resp:
+            raise Exception(str(resp))
+        return resp['content']
+
+    def test(self):
+        logger.info(f"[{__parser__}]:test: launching test query (getting logs from 10 days ago)",
+                    extra={'frontend': str(self.frontend)})
+
+        to = timezone.now()
+        since = to - datetime.timedelta(days=10)
+
+        try:
+            logs = self.get_logs(since=since, to=to)
+
+            return {
+                "status": True,
+                "data": logs
+            }
+        except Exception as e:
+            logger.exception(f"[{__parser__}]:test: {e}", extra={'frontend': str(self.frontend)})
+            return {
+                "status": False,
+                "error": str(e)
+            }
+
+    def execute(self):
+        since = self.last_api_call or (timezone.now() - datetime.timedelta(hours=24))
+        to = timezone.now()
+        logger.debug(f'since: {since}\n to: {to}\n', extra={'frontend': str(self.frontend)})
```

Comment:
DON'T put newlines in logs!
```suggestion
        logger.info(f'[{__parser__}]:execute: Getting logs from {since} to {to}', extra={'frontend': str(self.frontend)})
```